src/Test.UnitTests.Sarif/Baseline2/TrustMapTests.cs

```diff
@@ -0,0 +1,79 @@
+        public const string NameNoMatches = "NoMatchesFingerprint/v1";
+        public const string NameConstant = "ConstantFingerprint/v1";
+
+        [Fact]
+        public void TrustMap_Basics()
+        {
+            // Create a TrustMap for both of the Runs being compared
+            TrustMap beforeMap = new TrustMap();
+            TrustMap afterMap = new TrustMap();
+
+            // All properties are unknown to start
+            Assert.Equal(TrustMap.DefaultTrust, beforeMap.Trust(Set, NameGreat));
+
+            for (int i = 0; i < 20; ++i)
+            {
+                // Add a fingerprint which is always unique and all of which match
+                beforeMap.Add(new WhatComponent(Rule, Location1, Set, NameGreat, i.ToString()));
+                afterMap.Add(new WhatComponent(Rule, Location2, Set, NameGreat, i.ToString()));
+
+                // Add a fingerprint which is somewhat unique and always matches
+                beforeMap.Add(new WhatComponent(Rule, Location1, Set, NameMedium, (i % 3).ToString()));
+                afterMap.Add(new WhatComponent(Rule, Location2, Set, NameMedium, (i % 3).ToString()));
+
+                // Add a fingerprint which is always unique but has no matches
+                beforeMap.Add(new WhatComponent(Rule, Location1, Set, NameNoMatches, i.ToString()));
+                afterMap.Add(new WhatComponent(Rule, Location2, Set, NameNoMatches, (100 + i).ToString()));
+
+                // Add a fingerprint with constant value
+                beforeMap.Add(new WhatComponent(Rule, Location1, Set, NameConstant, "Constant"));
+                afterMap.Add(new WhatComponent(Rule, Location2, Set, NameConstant, "Constant"));
+            }
+
+            // Before comparing the runs, trust is based on uniqueness only
+            Assert.Equal(1.0f, beforeMap.Trust(Set, NameGreat));
+            Assert.Equal((3.0f / 20.0f), beforeMap.Trust(Set, NameMedium));
+            Assert.Equal(1.0f, beforeMap.Trust(Set, NameNoMatches));
+            Assert.Equal(0.0f, beforeMap.Trust(Set, NameConstant));
+
+            // Match the two runs to allow trust scoring based on matches
+            Assert.False(beforeMap.WasMatched);
+            Assert.False(afterMap.WasMatched);
+            afterMap.CountMatchesWith(beforeMap);
+            Assert.True(beforeMap.WasMatched);
+            Assert.True(afterMap.WasMatched);
+
+            // After comparing runs, trust is based on uniqueness and number of matches
+            // Note: Constants have no trust, unique and no match have low (default) trust.
+            Assert.Equal(1.0f, beforeMap.Trust(Set, NameGreat));
+            Assert.Equal((3.0f / 20.0f), beforeMap.Trust(Set, NameMedium));
+            Assert.Equal(TrustMap.DefaultTrust, beforeMap.Trust(Set, NameNoMatches));
+            Assert.Equal(0.0f, beforeMap.Trust(Set, NameConstant));
+
+            // Same scores from both maps
+            Assert.Equal(1.0f, afterMap.Trust(Set, NameGreat));
```

Comment:
>; [](start = 62, length = 1)

Consider writing these four tests as
```
Assert.Equal(beforeMap.Trust(Set, NameGreat), afterMap.Trust(Set, NameGreat));
```
for two reasons:
1. It better expresses the intent ("the two maps have the same scores")
2. It is DRYer; if you changed the values, you wouldn't have to remember to change them here too.

Reply:
Good idea.